draft-ietf-httpbis-priority.md

```diff
@@ -509,7 +509,7 @@ type PROTOCOL_ERROR.
 ~~~ drawing
 HTTP/2 PRIORITY_UPDATE Frame {
   Length (24),
-  Type (i) = 10,
+  Type (8) = 0x10,
```

Comment:
Thank you for filing the fix. Looks good to me, modulo the use of hexadecimal notation.

Regarding hexadecimal notation, I think we'd want to follow the decision made in https://github.com/httpwg/http2-spec/pull/1002?

Reply:
Agreed, no rush to merging this yet

Reply:
They merged, so we merge :)